scheduler/db/autoscaler_tables_postgres.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE app_scaling_specific_date_schedule
```

Comment:
what is this sql for?

Reply:
Hey @qibobo it's a convenience sql script that I used to get `mvn test` to run successfully. There's a couple tests in TestDataDbUtil.java that require the tables/columns in that script to exist.

Reply:
@mhottinger1 
We use [liquibase](https://github.com/cloudfoundry/app-autoscaler#initialize-the-database) to initialize the database. So I think the sql script is a duplication.

Reply:
dupe sql script removed per feedback.